pom.xml

```diff
@@ -1,263 +1,321 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<groupId>io.projectriff</groupId>
-	<artifactId>java-function-invoker</artifactId>
-	<version>0.1.5-SNAPSHOT</version>
-	<packaging>jar</packaging>
+    <groupId>io.projectriff</groupId>
+    <artifactId>java-function-invoker</artifactId>
+    <version>0.2.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
 
-	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.1.2.RELEASE</version>
-		<relativePath /> <!-- lookup parent from repository -->
-	</parent>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.2.0.BUILD-SNAPSHOT</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
 
-	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<java.version>11</java.version>
-		<spring-cloud-function.version>2.0.1.RELEASE</spring-cloud-function.version>
-		<skip.invoke>true</skip.invoke>
-	</properties>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>11</java.version>
+        <grpc.version>1.15.1</grpc.version>
```

Comment:
Please upgrade to [`1.23.0`](https://github.com/grpc/grpc-java/releases/tag/v1.23.0) as this release contains fixes for [HTTP2 vulnerabilities](https://www.kb.cert.org/vuls/id/605641/).

Reply:
Done with https://github.com/projectriff/java-function-invoker/pull/133/commits/9c4fa0b35e6903577609912a3d85a1cbbe223cb6.